README.md

```diff
@@ -1 +1,72 @@
-# WeSingAfrica-Capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project
+
+> This project contains the HTML, CSS and JavaScript Capstone Project. Capstone Projects are done at the end of a module at Microverse.
+
+# Project Name
+
+## WeSingAfrica Gospel Concert
+
+**WeSingAfrica Gospel Concert is a fictional annual worship concert that holds in Africa. Two editions have been held previously in Kenya and South Africa. This project showcases the Home and About Pages with sections relating to the event's programs, previous events gallery and invited guests.**
+
+![Image of the desktop view](./images/desktop-design.png)
+
+
+# Live Website
+
+Click [here](https://dammyshittu.github.io/WeSingAfrica-Capstone/) to view the website online.
+
+# This project was built with:
```

Comment:
- [ ] Nice to have loom video in the PR, let us include it in the Readme file as well.